src/infection-model/seir.tsx

```diff
@@ -272,15 +259,13 @@ export function getAllBracketCurves(inputs: CurveProjectionInputs) {
   ageGroupFatalityRates[ageGroupIndex.staff] = 0.026;
 
   // calculate R0 adjusted for housing type and capacity
-  let rateOfSpreadCells = R0Cells[rateOfSpreadFactor];
   const rateOfSpreadCellsAdjustment = 0.8; // magic constant
-  rateOfSpreadCells =
+  const rateOfSpreadCellsAdjusted =
     rateOfSpreadCells -
     (1 - facilityOccupancyPct) *
       (rateOfSpreadCells - rateOfSpreadCellsAdjustment);
-  let rateOfSpreadDorms = R0Dorms[rateOfSpreadFactor];
   const rateOfSpreadDormsAdjustment = 1.7; // magic constant
-  rateOfSpreadDorms =
+  const rateOfSpreadDormsAdjusted =
     rateOfSpreadDorms -
     (1 - facilityOccupancyPct) *
       (rateOfSpreadDorms - rateOfSpreadDormsAdjustment);
```

Comment:
@justkunz it's not strictly in scope for this change but I was wondering whether you think all these adjustments will still be valid when the input values come from an R(t) calculation?